### PR TITLE
Feature/prlc100 2

### PR DESCRIPTION
**Pull request by hmcts-abduldaim**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRLC100-2

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/common/common.content.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ const en = {
   ogl: 'All content is available under the <a class="govuk-link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated',
   errorSummaryHeading: 'There is a problem',
   saveAndSignOut: 'Save and sign out',
-  saveAsDraft: 'Save as draft',
+  saveAsDraft: 'Save and come back later',
```
[the remark this answers is not in the document]
Comment (channa-patil) [reply src/main/steps/common/common.content.ts:32]: @yogendraU I see this is common button label used everywhere.
@hmcts-abduldaim Let's add a new constant & label for c100.

File: src/main/steps/common/page.njk [hunk 2]
```diff
@@ -64,9 +67,17 @@
       {% endblock %}
     </div>
     <div class="govuk-grid-column-one-third">
-        {% block page_contact_us %}
```
Comment (channa-patil) [inline src/main/steps/common/page.njk:67]: @hmcts-abduldaim Again these are common changes, not sure why you removed dynamic page link & added hardcoded contact for help details.
Please revert, let's try reuse the same.

File: src/main/steps/edge-case/dashboard/template.njk [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {% extends "common/template.njk" %}
 
 {% block page_content %}
-<a href="/respondent/task-list" class="govuk-link">Respondent</a>
```
Comment (channa-patil) [inline src/main/steps/edge-case/dashboard/template.njk:4]: @hmcts-abduldaim Lets not remove any respondent existing links please.
We just need to add new changes for c100 without altering respondent journey.

**Review by channa-patil — CHANGES_REQUESTED** [review]
Please create PR against C100 base branch
https://github.com/hmcts/prl-citizen-frontend/tree/feature/PRL-1615-c100-rebuild-journey

**Pull request closed without merging**